docs/config.md

```diff
@@ -448,6 +448,11 @@ branches. Since the local branch isn't created, the remote branch will be
 deleted if you push the branch with `jj git push --branch` or `jj git push
 --all`.
 
+### Prefix for generated branches on push
+
+`jj git push --change` generates branch names with a prefix of "push-" by
+default. You can pick a different prefix by setting `push.branch-prefix`.
```

Comment:
Not for this PR: the inconsistency in naming of `git.auto-local-branch` and `push.branch-prefix` is mildly annoying.

Perhaps they could be `branch.push-prefix` and `branch.auto-create-local`, or something along these lines?

Reply:
Good point. It probably belongs under `[git]`.